docs/quick-start-v1alpha2.md

```diff
@@ -0,0 +1,114 @@
+# Quick Start (for V1alpha2)
+
+## Installing v1alpha2 CRD and operator
+
+We recommend to deploy Kubeflow in order to use the TFJob v1alpha1 operator. While v1alpha2 is still in experimental, we recommend to run the operator locally. Please have a look at [developer_guide.md](../developer_guide.md)
```

Comment:
Why are we suggesting users run the v1alpha2 operator locally? If we want users to try v1alpha2 why can't we make it easy to deploy the v1alpha2 operator as part of Kubeflow deployment?

Reply:
Because we do not have the docker image and release process for v1alpha2 now. Then I think it is better to use it locally.

Reply:
What's the point of adding it to the docs and asking users to try it until we have those things? Running locally might be suitable for devs but for users we should work on fixing those things first.

Reply:
Sorry for the lack of background. There are some people in China who are not interested in the development but they want to have a try on v1alpha2. Then I decided to wrote such a doc.

Reply:
Got it thanks. How about changing the docs to make it clear this is only for early adopters e.g.

Change Quick Start (v1alpha2) to something like

# Testing v1alpha2

v1alpha2 is not ready yet but if you still want to try it out you can run it locally by following the instructions below. You need to run it locally because it is not included in Kubeflow yet.

Reply:
sgtm